beacon-node-oapi.yaml

```diff
@@ -97,6 +97,8 @@ paths:
     $ref: "./apis/beacon/pool/sync_committees.yaml"
   /eth/v1/beacon/pool/voluntary_exits:
     $ref: "./apis/beacon/pool/voluntary_exists.yaml"
+  /eth/v1/beacon/pool/validator_registration:
```

Comment:
Where should this endpoint be, don't know if it fits well into pool 

could create a new folder called builder and it would be 

/eth/v1/beacon/builder/validator_registration

depends on what the eth guys want